README.md

```diff
@@ -74,7 +74,17 @@ Include the `INSTALLATION_TYPE`. See [here](#variables-table) about this and oth
 INSTALLATION_TYPE=<managed/managed-api> make cluster/prepare/local
 ```
 
-### 3. Run integreatly-operator
+### 3. Create a cr (optional)
+
+If you are running RHOAM against a cluster which is smaller than the requirements listed above, you 
+should use the IN_PROW variable, otherwise the installation will not complete. 
+If you have a cluster which meets the requirements, please skip this step.
+
+```shell script
+INSTALLATION_TYPE=managed-api IN_PROW=true make deploy/integreatly-rhmi-cr.yml
```

Comment:
https://github.com/integr8ly/integreatly-operator/blob/master/README.md#variables-table at the beginning (line 111 - have no idea how to link the readme lines of code, sorry) of the section I've mentioned creating CR, but it's obvious that nobody can find it there. Wouldn't it be better to more that "section" here instead? So not only IN_PROW variable but all others will be honoured. And, maybe, link the table. Also, in case we are moving section, we shall update/remove the link from `Consider using IN_PROW [optional variable](#variables-table) in case of limited resources.` in here  https://github.com/integr8ly/integreatly-operator/blob/master/README.md#local-development

Reply:
Yes, make sense.

Reply:
https://github.com/integr8ly/integreatly-operator#2-prepare-your-cluster and here as well in line `See [here](#variables-table) about this and other optional configuration variables.` <- we won't need that line then as the "optional configuration" will be just after the section